src/main/java/com/box/sdk/BoxWebHook.java

```diff
@@ -528,6 +528,8 @@ void parseJSONMember(JsonObject.Member member) {
                 assert false : "A ParseException indicates a bug in the SDK.";
             } catch (MalformedURLException e) {
                 assert false : "A MalformedURLException indicates a bug in the SDK.";
+            } catch (Exception e) {
```

Comment:
You should remove the `catch` blocks above this so all exceptions get transformed into a `BoxDeserializationException`